src/Symfony/Component/Notifier/Bridge/Mercure/.gitattributes

```
@@ -0,0 +1,4 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/phpunit.xml.dist export-ignore
+/Tests export-ignore
```

```suggestion
/Tests export-ignore
/phpunit.xml.dist export-ignore
/.gitattributes export-ignore
/.gitignore export-ignore
```

Please change the order to stay consistent with the other bridges